tests/integration/test.sync.js

```diff
@@ -332,11 +332,11 @@ adapters.forEach(function (adapters) {
       replications.cancel();
     });
 
-    it.skip('Test sync cancel called twice', function (done) {
+    it('Test sync cancel called twice', function (done) {
       var db = new PouchDB(dbs.name);
       var remote = new PouchDB(dbs.remote);
       var replications = db.sync(remote).on('complete', function () {
-        setTimeout(done); // let cancel() get called twice before finishing
+        done();
```

Comment:
I checked the code and can't see why we would need to do `setTimeout()`. The second `cancel()` call should synchronously do nothing, since it looks like this:

```js
Sync.prototype.cancel = function () {
  if (!this.canceled) {
    this.canceled = true;
    this.push.cancel();
    this.pull.cancel();
  }
};
```

I felt comfortable re-enabling this test because it looks exactly the same as the test that precedes it, if you consider that the second `cancel()` is necessarily a no-op. I might have mistakenly thought before that it needed a `setTimeout()`, but seems to me now that it doesn't.